### DISCUSSION TITLE
how can i download BitcrazeVM.ova

### DISCUSSION THREAD

**Discussion by vanbong** [bitcraze/discussions#47] [Q&A]
bong@bong-IRH-Gaming:~/bitcraze-vm-master$ ./createBitcrazeVM.sh 
Running packer...
OVA file not found. Starting VM creation...
virtualbox-iso: output will be in this color.

==> virtualbox-iso: Retrieving ISO
==> virtualbox-iso: Trying http://archive.ubuntu.com/ubuntu/dists/focal/main/installer-amd64/current/legacy-images/netboot/mini.iso
==> virtualbox-iso: Trying http://archive.ubuntu.com/ubuntu/dists/focal/main/installer-amd64/current/legacy-images/netboot/mini.iso?checksum=md5%3Ae5bf713621b43207dead707396da29f6
==> virtualbox-iso: http://archive.ubuntu.com/ubuntu/dists/focal/main/installer-amd64/current/legacy-images/netboot/mini.iso?checksum=md5%3Ae5bf713621b43207dead707396da29f6 => /home/bong/.cache/packer/f3e26dd0903967b46f46bd5e981f05186aae16e3.iso
Build 'virtualbox-iso' errored after 173 milliseconds 669 microseconds: Output directory exists: output-virtualbox-iso

Use the force flag to delete it prior to building.

==> Wait completed after 173 milliseconds 726 microseconds

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Output directory exists: output-virtualbox-iso

Use the force flag to delete it prior to building.

==> Builds finished but no artifacts were created.
Starting VM provisioning...
virtualbox-ovf: output will be in this color.

==> virtualbox-ovf: Retrieving Guest additions
==> virtualbox-ovf: Trying /usr/share/virtualbox/VBoxGuestAdditions.iso
==> virtualbox-ovf: Trying /usr/share/virtualbox/VBoxGuestAdditions.iso
==> virtualbox-ovf: /usr/share/virtualbox/VBoxGuestAdditions.iso => /usr/share/virtualbox/VBoxGuestAdditions.iso
==> virtualbox-ovf: Retrieving OVF/OVA
==> virtualbox-ovf: Trying output-virtualbox-iso/BitcrazeVM.ova
==> virtualbox-ovf: Trying output-virtualbox-iso/BitcrazeVM.ova
==> virtualbox-ovf: Download failed source path error: stat /home/bong/bitcraze-vm-master/output-virtualbox-iso/BitcrazeVM.ova: no such file or directory
==> virtualbox-ovf: error downloading OVF/OVA: [source path error: stat /home/bong/bitcraze-vm-master/output-virtualbox-iso/BitcrazeVM.ova: no such file or directory]
==> virtualbox-ovf: Deleting output directory...
Build 'virtualbox-ovf' errored after 545 milliseconds 993 microseconds: error downloading OVF/OVA: [source path error: stat /home/bong/bitcraze-vm-master/output-virtualbox-iso/BitcrazeVM.ova: no such file or directory]

==> Wait completed after 546 milliseconds 86 microseconds

==> Some builds didn't complete successfully and had errors:
--> virtualbox-ovf: error downloading OVF/OVA: [source path error: stat /home/bong/bitcraze-vm-master/output-virtualbox-iso/BitcrazeVM.ova: no such file or directory]

==> Builds finished but no artifacts were created.

**Comment by knmcguire** [comment]
Hi! I don't think you have read [the readme of the bitcraze-vm repo](https://github.com/bitcraze/bitcraze-vm#downloading) clearly. You can download a [precompiled VM from the release page](https://github.com/bitcraze/bitcraze-vm/releases). What you are currently doing is remaking everything, which is mostly meant for releases and for at bitcraze to use internally and you shouldn't have to do that.
